### ISSUE TITLE
Default project folder is undefined

### ISSUE THREAD

**Issue by CodingItWrong** [gossi/ember-cli-create#12]
In `0.0.3` when I create a project the default folder name is "undefined", instead of the project name I entered:

```
~/apps/temp # ember-cli-create octane-test
? What kind of ember project you want to create? Ember App
  -> Regular ember app wo/ welcome page
? Where to create your project?: (./undefined)
```

I think this worked in `0.0.2`; the default folder name would have been `octane-test` instead of `undefined`.

**Comment by CodingItWrong** [comment]
It seems like the project name in `package.json` is set to `undefined` as well; so it seems like the project name parameter is just lost somehow.

**Comment by gossi** [comment]
Thanks, good catch. Fixed in `v0.0.4` 😃 🌴 🎄
